database/nakadi/10_data/04_tables/02_subscription.sql

```diff
@@ -0,0 +1,11 @@
+CREATE TABLE IF NOT EXISTS zn_data.subscription (
+  s_id varchar(255) NOT NULL PRIMARY KEY CHECK (s_id <> ''),
+  s_subscription_object jsonb NOT NULL,
+  CHECK (s_subscription_object->>'id' = s_id)
```

Comment:
We need a key-value store. By the time I created the event-type table, I did provided an extra column for the `name` because I did not know that postgres offers some consistency checks in json objects properties. But I see no harm in using the same approach. We could in the future remove both columns from both tables and have it implemented in a consistent way across the project.